tests/test_diaPipe.py

```diff
@@ -292,6 +293,18 @@ def check_diaObjects(bbox, wcs, diaObjects):
         # Verify that no objects inside the bounding box were removed
         self.assertEqual(nIn1, nIn0)
 
+    def test_make_empty_catalog(self):
+        """Check that an empty catalog has the correct format.
+        """
+        tableName = "DiaObject"
+        config = self._makeDefaultConfig(config_file=self.config_file.name)
+        diaPipeTask = DiaPipelineTask(config=config)
+        emptyDiaObjects = make_empty_catalog(diaPipeTask.schema, tableName=tableName)
```

Comment:
If you provide a test schema directly, then there's no need to mix `DiaPipelineTask` into the test...

Reply:
That is true, though here I wanted to also test that `diaPipeTask.schema` is getting set correctly. That's why I added this test to `test_diaPipe` instead of creating a `test_utils`, but I am happy to move it out and just make the schema directly.

Reply:
I'm not sure that's actually being tested here -- the only thing you compare the schema to is itself.

Reply:
OK, I will move it to `test_utils`, and also add a check of a few of the values.